testsuite/integration-deprecated/src/test/java/org/keycloak/testsuite/broker/AbstractFirstBrokerLoginTest.java

```diff
@@ -355,61 +324,6 @@ public void testLinkAccountByEmailVerificationTwice() throws Exception {
         Assert.assertThat(infoPage.getInfo(), startsWith("You successfully verified your email. Please go back to your original browser and continue there with the login."));
     }
 
-    /**
-     * Tests that duplication is detected and user wants to link federatedIdentity with existing account. He will confirm link by email
-     */
-    @Test
-    public void testLinkAccountByEmailVerificationDifferentBrowser() throws Exception, Throwable {
```

Comment:
Removed in favor of org.keycloak.testsuite.broker.AbstractBrokerTest#testVerifyEmailInNewBrowserWithPreserveClient.